packages/server/controllers/guilds.js

```diff
@@ -82,6 +82,41 @@ async function getGuildMembers(guildId) {
   return members;
 }
 
+async function getGuildsForUser(userId) {
+  try {
+    const user = await prisma.users.findFirst({
+      where: {
+        userId: userId,
+      },
+    });
+
+    if (!user) {
+      throw new Error(`User with ID ${userId} not found`);
+    }
```

Comment:
These checks won't be necessary. I'm getting all of us to migrate over to express-validator to streamline error handling. I'll tell you more details in Discord but you can start by reading docs [here](https://express-validator.github.io/docs/guides/getting-started#adding-a-validator).